ansible/roles/app-role-k8s/files/opt/app/current/bin/node/k8s.sh

```diff
@@ -693,6 +704,10 @@ updateApiserverCerts() {
   runKubeadm init phase certs apiserver
 }
 
+annotateMyNode(){
+  runKubectl annotate no $(getMyNodeName) node.beta.kubernetes.io/instance-id="$MY_INSTANCE_ID" --overwrite=true
```

Comment:
我试了下不需要这个 `--overwrite=true`，这种尽量能不用的就不用，避免误操作改了用户的数据。

Reply:
如果该 annotate 已经存在的话，再次执行这个会报错，--overwrite=true 主要是为了实现幂等

Reply:
这个只在 initCluster 的时候执行一次，如果失败肯定是有问题，需要上去看一眼为什么出错。

Reply:
```
$ kubectl --kubeconfig /data/kubernetes/admin.conf annotate no master1 node.beta.kubernetes.io/instance-id=i-5ul9qrwq
error: --overwrite is false but found the following declared annotation(s): 'node.beta.kubernetes.io/instance-id' already has a value (i-5ul9qrwq)

$ kubectl annotate -h
--overwrite=false: If true, allow annotations to be overwritten, otherwise reject annotation updates that
overwrite existing annotations
```

Reply:
这是创建集群时候执行的吗

Reply:
`initCluster` 只在创建集群的时候执行一次

Reply:
好的，我去掉 --overwrite=true 该参数